MVVM Module/Tests/Feed UI/Helpers/FeedViewController+TestHelpers.swift

```diff
@@ -40,6 +40,18 @@ extension FeedViewController {
 		ds?.tableView?(tableView, cancelPrefetchingForRowsAt: [index])
 	}
 
+	var errorView: ErrorView? {
+		tableView.tableHeaderView as? ErrorView
+	}
+
+	func simulateTapOnErrorMessage() {
+		errorView?.button.simulateTap()
+	}
+
+	var errorMessage: String? {
+		return errorView?.message
+	}
+
```

Comment:
Since `errorView` getter is public in production, there's no need to redefine it here
Also there's no need to put question marks after `errorView` since it has a force unwrapped type `ErrorView!`

```suggestion
	func simulateTapOnErrorMessage() {
		errorView.button.simulateTap()
	}

	var errorMessage: String? {
		return errorView.message
	}

```

Reply:
Done